index.html

```diff
@@ -484,14 +488,146 @@ <h4><code>readproperty</code></h4>
         </pre>
       </section>
     </section>
-    <section id="actions">
-      <h3>Actions</h3>
-    </section>
-    <section id="events">
-      <h3>Events</h3>
-    </section>
-    <section id="events">
-      <h3>Other Messages</h3>
+
+    <section id="writeproperty">
+      <h3><code>writeproperty</code></h3>
+      <section id="writeproperty-request">
+        <h4>Request</h4>
+        <p>To set the value of a <a>Property</a> of a <a>Thing</a>, a <a>Consumer</a> MUST send a <a
+            href="#websocket-messages">message</a> to the <a>Thing</a> which contains the following members:</p>
+        <table class="def">
+          <caption>Members of a <code>writeproperty</code> request message</caption>
+          <thead>
+            <tr>
+              <th>Member</th>
+              <th>Type</th>
+              <th>Assignment</th>
+              <th>Description</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td><code>messageType</code></td>
+              <td>string</td>
+              <td>"request"</td>
+              <td>A string which denotes that this message is a request sent from a Consumer to a Thing.</td>
+            </tr>
+            <tr>
+              <td><code>operation</code></td>
+              <td>string</td>
+              <td>"writeproperty"</td>
+              <td>A string which denotes that this message relates to a <code>writeproperty</code> operation.</td>
+            </tr>
+            <tr>
+              <td><code>name</code></td>
+              <td>string</td>
+              <td>Mandatory</td>
+              <td>The name of the <a>Property</a> whose value should be set, as per its key in the
+                <code>properties</code> member of the <a>Thing Description</a>.
+              </td>
+            </tr>
+            <tr>
+              <td><code>value</code></td>
+              <td>any</td>
+              <td>Mandatory</td>
+              <td>The desired new value of the <a>Property</a>, with a format conforming to the data schema of the
```

Comment:
format reads weird here but might be ok. Maybe something more like "structure" is better?

Reply:
I changed "format" to "type and structure" (updated for the `readproperty` operation too). Let me know if think that's still not quite right.